docs/.readthedocs.yaml

```diff
@@ -12,12 +12,12 @@ build:
     python: "3.11"
   jobs:
     post_checkout:
-      - python3 docs/build_requirements.py
+      - python3 .sphinx/build_requirements.py
```

Comment:
I think this should be:
```suggestion
      - python3 docs/.sphinx/build_requirements.py
```

As that's where I'm seeing that file and the build is currently broken on that file;
![image](https://github.com/canonical/ubuntu-pro-for-wsl/assets/11138291/f4e483f9-3fbf-41ff-9b5a-330239e44616)